internal/rbd/controllerserver.go

```diff
@@ -833,6 +878,48 @@ func (cs *ControllerServer) DeleteVolume(
 	}
 	defer cs.OperationLocks.ReleaseDeleteLock(volumeID)
 
+	if isMigrationVolHandle(volumeID) {
+		log.DebugLog(ctx, "migration volume handle : %s", volumeID)
+		pMH, pErr := parseMigrationVolHandle(volumeID)
+		if pErr != nil {
+			log.DebugLog(ctx, "failed to parse migration volume handle: %s , err: %v", volumeID, pErr)
+
+			return nil, pErr
+		}
+		rv := &rbdVolume{}
+		if pMH != nil {
```

Comment:
why this check is needed? in any case both error and nil values are returned from parseMigrationVolHandle?

Reply:
placed it for more safety as there is a dereferencing in place, but removed it in current refresh of the patchset.